### PR TITLE
Notification bar

### LINKED ISSUE #276: Implement Notification Popup Component
https://projects.invisionapp.com/d/main/default/#/console/13241316/281903494/preview

### PR DESCRIPTION
**Pull request by tanx**
Closes #276

### CODE REVIEW

File: src/action/grpc.js [hunk 1]
```diff
@@ -69,11 +69,8 @@ class GrpcAction {
     return new Promise((resolve, reject) => {
       listen = method ? `${listen}_${method}` : listen;
       this._ipcRenderer.once(listen, (e, arg) => {
-        if (arg.err) {
-          log.error('GRPC: Error from method', method, arg.err);
-          return reject(arg.err);
-        }
-        resolve(arg.response);
+        if (arg.err) reject(arg.err);
```
Comment (valentinewallace) [inline src/action/grpc.js]: nice!!

File: src/action/grpc.js [hunk 2]
```diff
@@ -69,11 +69,8 @@ class GrpcAction {
...
+        else resolve(arg.response);
```
Comment (valentinewallace) [inline src/action/grpc.js]: nice!

File: src/component/button.js [hunk 3]
```diff
@@ -212,6 +212,52 @@ SmallButton.propTypes = {
...
+
+const smallPillStyles = StyleSheet.create({
+  touchable: {
+    paddingTop: 3,
+    paddingBottom: 4,
+    paddingLeft: 15,
+    paddingRight: 15,
+  },
+  text: {
+    fontFamily: 'OpenSans Regular',
```
Comment (valentinewallace) [inline src/component/button.js]: Noticed this text style is used in multiple places, might be good to make a <SmallText> component in text.js?

File: stories/component/notification.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { NotificationBar } from '../../src/component/notification';
+import NotificationAction from '../../src/action/notification';
+import { Store } from '../../src/store';
+
+const store = new Store();
+export const notify = new NotificationAction(store);
+
+storiesOf('Notification', module).add('Notification Bar', () => (
```
Comment (valentinewallace) [inline stories/component/notification.js]: Looks cute

File: src/component/button.js (hunk 3 above)
Comment (tanx) [reply src/component/button.js]: In general it’s good to reuse Text components within larger view components. Buttons are pretty atomic components and have pretty specific styles. So it’s probably more maintainable to specify those here directly.

**Comment by tanx** [comment]
Awesome! Thanks for the review :)

**Pull request merged**